lib/Pretty_printing.ml

```diff
@@ -213,7 +205,8 @@ and pretty_print_transformed_type st trans =
   in
   match trans with
   | Identity -> pretty_print_sizedtype st
-  | Lower _ | Upper _ | LowerUpper _ | OffsetMultiplier _ ->
+  | Lower _ | Upper _ | LowerUpper _ | Offset _ | Multiplier _
+   |OffsetMultiplier _ ->
```

Comment:
Space-vbar should be transposed so it lines up and is `| OffsetMultiplier`

Reply:
Sadly, this is what OCamlformat makes of it.